src/ducks/Studio/AdvancedView/Histogram/index.js

```diff
@@ -31,13 +33,21 @@ function rangeFormatter ([left, right]) {
   return usdFormatter(left) + ' - ' + usdFormatter(right)
 }
 
-const Frame = ({ range, value, ticker, width }) => {
+const Frame = ({ range, value, ticker, width, price, isRed }) => {
```

Comment:
@DmitriVanGuard Millify we are doing inside this func, why than we can't calculate red color using price and range fields. And make less interface.

Reply:
Addressed issue was fixed in the new commits